meta/schemas/wifi.yaml

```diff
@@ -0,0 +1,88 @@
+$id: https://openwrt.org/schema/wifi.yaml
+title: WiFi
+description: WiFi hardware description
+type: object
+
+required:
+  - freq_ghz
+  - hardware
+  - driver
+
+examples:
+  - driver: unknown
+    freq_ghz: 2.4
```

Comment:
What about having list of supported standards like 802.11b, 802.11g etc. instead? Having information about MIMO features would be nice as well, using the popular `TxR:S` notation

* `T` Number of transmit radio chains
* `R` Number of receive radio chains
* `S` Number of spatial data streams